src/local-item.ts

```diff
@@ -8,7 +7,22 @@ type Item = {
   content: string;
 };
 
-const items = new Map<string, Item>();
+const items = new Map<any, any>();
+
+export function getItem(req: Request, res: Response) {
+  addHeadersToResponse(res);
+  if (req.query.itemId) {
+    const toReturn = items.get(req.query.itemId);
+    toReturn.itemId = req.query.itemId;
+    return res.status(200).json(toReturn);
+  } else {
+    const reducedItems: ({ itemId: string } & Item)[] = [];
+    items.forEach((value: Item, key: string) => {
+      reducedItems.push({ itemId: key, ...value });
+    })
```

Comment:
let's add a comment line here:

// return the list of all items

Reply:
ok

Reply:
done